packages/astro/src/vite-plugin-jsx/tag.ts

```diff
@@ -18,9 +16,6 @@ export default async function tagExportsWithRenderer({
 	rendererName: string;
 	root: URL;
 }): Promise<PluginObj> {
-	const astroServerPath = fileURLToPath(
-		await importMetaResolve('astro/server/index.js', root.toString())
-	);
```

Comment:
This is not needed anymore as I made a change to dedupe `astro`, which means we can refer to the string unresolved. We do the same for Astro compile too:

https://github.com/withastro/astro/blob/4e37e82b4f4623e99bd58ebd70b5da0879d1713b/packages/astro/src/core/compile/compile.ts#L46